apps/acrs/behaviours/resume-form-session.js

```diff
@@ -0,0 +1,203 @@
+/* eslint-disable consistent-return */
+'use strict';
+
+const axios = require('axios');
+const config = require('../../../config');
+const moment = require('moment');
+const baseUrl = `${config.saveService.host}:${config.saveService.port}/saved_applications/email/`;
+const _ = require('lodash');
+
+const encodeEmail = email => Buffer.from(email).toString('hex');
+
+const SESSION_DEFAULTS = config.sessionDefaults;
+
+module.exports = superclass => class extends superclass {
+  cleanSession(req) {
+    let cleanList = Object.keys(req.sessionModel.attributes);
+    const leaveFields = SESSION_DEFAULTS.fields.concat(['csrf-secret', 'user-cases']);
+    cleanList = cleanList.filter(item => leaveFields.indexOf(item) === -1);
+
+    req.sessionModel.unset(cleanList);
+    req.sessionModel.set('steps', SESSION_DEFAULTS.steps);
+  }
+  // GET lifecycle
+  configure(req, res, next) {
+    // reset session but keeping login fields, csrf and preexisting cases garnered from API
+    this.cleanSession(req);
+    // skip requesting data service api when running in local and test mode
+    if (config.env === 'local' || config.env === 'test') {
+      return super.configure(req, res, next);
+    }
+    return this.getCases(req, res, next);
+  }
+
+  getCases(req, res, next) {
+    const email = req.sessionModel.get('user-email');
+    const idType = req.sessionModel.get('id-type');
+
+    return axios.get(baseUrl + encodeEmail(email))
+      .then(response => {
+        const sessionCases = req.sessionModel.get('user-cases') || [];
+        const unsubmittedCases = _.filter(response.data, record => !record.submitted_at &&
+                                                                    record.session['id-type'] === idType);
```

Comment:
I would have thought a user would want to see all the applications on the page (uan and brp) but that the radio labels should say if its a brp login or a uan login based on `idType`

Reply:
That's not the case, if that stays that way we will have undefined value for uan as it will be null  best is to filter the number depends on what the applicant used. Also would have suggested same

Reply:
This will show saved applications though - so each referral will have an 'id-type' value in session. From that I reckon you can decide for each radio button whether to show the UAN or BRP in the label

Reply:
@TemitopeAyokuHO will you ask product manager whether the page should show BRP results, UAN results or all together?